### ISSUE TITLE
mdc-drawer-ink-color($color) not working

### ISSUE THREAD

**Issue by MrChriZ** [material-components/material-components-web#2670]
### What MDC Web Version are you using?

0.35.0

### What browser(s) is this bug affecting?

All

### What OS are you using?

Linux

### What are the steps to reproduce the bug?

Use SCSS
@import 'node_modules/material-components-web/material-components-web.scss';

```
#temporaryDrawer{
  @include mdc-drawer-fill-color(#2cd46c);
  @include mdc-drawer-ink-color(red);
}

 <aside id="temporaryDrawer" class="mdc-drawer mdc-drawer--temporary demo-drawer mdc-drawer--open">
....
</aside>
```
React code here:
https://github.com/MrChriZ/reactMDCTest

Demo:
https://mrchriz.github.io/reactMDCTest/

### What is the expected behavior?
Drawer text turns red.

### What is the actual behavior?
Drawer background colour is correct.
Drawer ink colour is not correct.

**Comment by bonniezhou** [comment]
Thanks for filing this bug!

This is happening because the default `mdc-list` also sets the ink color of the list items, which overrides the `mdc-drawer` mixin because of its proximity to the list item.

Seems reasonable to make `mdc-list` inherit its ink color by default. If it's in the body and no ink color is set, it would inherit the `on-background`/`on-surface` from the theme. Added to tracker.

**Comment by bonniezhou** [comment]
Hey @RhysyG sorry for the delayed response.

We provide `mdc-drawer-ink-color` as a public mixin so users can easily customize the ink color without having to think about which CSS properties to change. `mdc-theme-prop` is an implementation detail of `mdc-drawer-ink-color`.

We try to use our public mixins as much as possible to showcase its usage, which is why it's used in the `.mdc-drawer--permanent` styles.

**Comment by RhysyG** [comment]
@bonniezhou ,

No worries, thanks for answering, makes sense!

Cheers.

**Comment by abhiomkar** [comment]
Fixed in #3417
